GitTfs.Vs2015/GitTfs.Vs2015.csproj

```diff
@@ -139,7 +139,7 @@
     </When>
   </Choose>
   <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.5'">
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v4.5' Or $(TargetFrameworkVersion) == 'v4.5.2')">
```

Comment:
The way to detect an installed instance of Visual Studio 2017 required a NuGet package reference (though, this project uses Paket), and that package required .Net 4.5.2.